recipes/cas-offinder/meta.yaml

```diff
@@ -0,0 +1,29 @@
+package:
+  name: cas-offinder
+  version: 2.4
+
+source:
+  url: https://github.com/snugel/cas-offinder/archive/5a5d24922005db5ab47dc0aceb0572a00b8f0e27.tar.gz
+  sha256: 7946039383fb826618959b6c729c8322355db59e2303ebaf1f2883f0365c103c
+
+requirements:
+  build:
+    - cmake 
+    - pocl #[linux]
+    - libgcc # [linux]
+    - llvm # [osx]
+  run:
+    - cmake 
+    - pocl # [linux]
+    - libgcc # [linux]
+test:
+  commands:
+    - cas-offinder -h
+
+about:
+  home: https://github.com/snugel/cas-offinder
+  license: BSD
```

Comment:
Can you make that
```yaml
  license: BSD-3-Clause AND MIT
```
? References: 
https://github.com/snugel/cas-offinder/blame/5a5d24922005db5ab47dc0aceb0572a00b8f0e27/README.md#L321
https://github.com/snugel/cas-offinder/blob/5a5d24922005db5ab47dc0aceb0572a00b8f0e27/ext/dirent.h#L4-L23

Reply:
It's all good. I really appreciate what you have done to help me.